### PR TITLE
Fix forwarding policy identifier cleanup in PFCP handlers

### PR DESCRIPTION
**Pull request by sergeymatov**

### CODE REVIEW

File: upf/pfcp.c [hunk 1]
```diff
@@ -1449,6 +1450,13 @@ encode_forwarding_policy (void *p, u8 ** vec)
   return 0;
 }
 
+void
+free_forwarding_policy (void *p)
```
Comment (RoadRunnr) [inline upf/pfcp.c:1454]: the type of the IE must also be changed to `SIMPLE_IE_FREE` for this to take effect.

**Review by RoadRunnr — CHANGES_REQUESTED** [review]

File: upf/pfcp.c (hunk 1 above)
Comment (sergeymatov) [reply upf/pfcp.c:1454]: Fixed

File: upf/pfcp.c [hunk 2]
```diff
@@ -1434,6 +1434,7 @@ decode_forwarding_policy (u8 * data, u16 length, void *p)
     return PFCP_CAUSE_INVALID_LENGTH;
 
   get_vec (v->identifier, length, data);
+  length = 0;
```
[the remark this answers is not in the document]
Comment (sergeymatov) [reply upf/pfcp.c:1437]: Done

**Review by korroot — APPROVED** [review]
LGTM

**Review by RoadRunnr — APPROVED** [review]

**Pull request merged**